### PR TITLE
Removed mentions of cloud-based devices.

### LINKED ISSUE #1280: Co-Management
Can you expand into CAS supported client numbers when co-management is enabled?  https://docs.microsoft.com/en-us/sccm/core/plan-design/configs/size-and-scale-numbers#bkmk_cas

How does this affect client numbers? Are co-managed devices considered "cloud devices"? I quote: "For example, in a hierarchy you can support 700,000 desktops, up to 25,000 Mac and Windows CE 7.0 devices, and up to 300,000 cloud-based devices when you integrate Microsoft Intune"  

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9c595a86-976c-dc2c-5b01-fb9d6b761f17
* Version Independent ID: a90f5193-559b-07b0-519a-d5ef32de3151
* Content: [Size and scale - Configuration Manager](https://docs.microsoft.com/en-us/sccm/core/plan-design/configs/size-and-scale-numbers#bkmk_cas)
* Content Source: [sccm/core/plan-design/configs/size-and-scale-numbers.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/core/plan-design/configs/size-and-scale-numbers.md)
* Product: **configuration-manager**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

### PR DESCRIPTION
**Pull request by ChrisKibble**
Fixes #1280

### CODE REVIEW

**Comment by PRMerger9** [comment]
@ChrisKibble : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sccm/core/plan-design/configs/size-and-scale-numbers.md [hunk 1]
```diff
@@ -177,13 +177,7 @@ A central administration site supports a total number of devices that includes u
 
 - 25,000 devices that run Mac and Windows CE 7.0  
 
-- One of the following, depending on how your deployment supports mobile device management (MDM):  
-
-    - 100,000 devices that you manage by using on-premises MDM  
-
-    - 300,000 cloud-based devices  
-
-For example, in a hierarchy you can support 700,000 desktops, up to 25,000 Mac and Windows CE 7.0 devices, and up to 300,000 cloud-based devices when you integrate Microsoft Intune. This hierarchy supports a total of 1,025,000 devices. If you support devices that are managed by on-premises MDM, the total for this hierarchy is 825,000 devices.
```
Comment (aczechowski) [inline sccm/core/plan-design/configs/size-and-scale-numbers.md]: I suggest keeping the example, just editing out the cloud-based devices part (and then adjusting the math accordingly 😉)

File: sccm/core/plan-design/configs/size-and-scale-numbers.md [hunk 2]
```diff
@@ -208,13 +202,7 @@ A stand-alone primary site supports the following number of devices:
 
     - 25,000 devices that run Mac and Windows CE 7.0
 
-    - One of the following, depending on how your deployment supports mobile device management:  
-
-        - 50,000 devices that you manage by using on-premises MDM  
-
-        - 150,000 cloud-based devices  
-
-For example, a stand-alone primary site that supports 150,000 desktops and 10,000 Mac or Windows CE 7.0 can support only an additional 15,000 devices. Those devices can be either cloud-based or managed by using on-premises MDM.
```
Comment (aczechowski) [inline sccm/core/plan-design/configs/size-and-scale-numbers.md]: Same here, keep the example, edited

**Comment by aczechowski** [comment]
I'm working on this article for broader hybrid deprecation changes. I'm going to be a jerk and just close this PR, since I already have it captured with the other changes. Sorry 😢

**Pull request closed without merging**